exdb/views.py

```diff
@@ -368,6 +368,97 @@ def get_context_data(self, *args, **kwargs):
         return context
 
 
+class CompletionBoardView(TemplateView):
+    access_level = 'basic'
+    template_name = 'exdb/completion_board.html'
+
+    def get_context_data(self, *args, **kwargs):
+        context = super(CompletionBoardView, self).get_context_data(*args, **kwargs)
+
+        if not self.request.user.is_hallstaff():
```

Comment:
Since there's currently no other options for `access_level`, I think we'll come back to this later.